README.md

```diff
@@ -41,19 +40,19 @@ The key/value pairs insides this ConfigMap are:
 
 The reference's location and format depends on the following scenarios:
 
-1. OLM-enabled Operator: Use the `statusDescriptor` part of the CSV to mark which `status` properties reference the binding data:
+1. OLM-enabled Operator: Use the `statusDescriptor` part of the CSV to mark which `status` properties reference the [binding data](#service-binding-schema):
```

Comment:
How about `specDescriptor` for getting data from `spec` fields?

Reply:
that's an interesting idea, I hadn't thought the spec part.   Makes sense.

Reply:
Added.